nautobot/docs/additional-features/jobs.md

```diff
@@ -333,6 +333,16 @@ These two methods will load data in YAML or JSON format, respectively, from file
 
     ![Adding the run action to a permission](../../media/admin_ui_run_permission.png)
 
+It is a key concept to understand the 3 `class_path` elements:
```

Comment:
Maybe add a sub-heading here so that the "see above" references below can actually link directly to it?

```suggestion
### Jobs and `class_path`

It is a key concept to understand the 3 `class_path` elements:
```